crates/common/log_manager/Cargo.toml

```diff
@@ -11,13 +11,13 @@ repository = { workspace = true }
 [dependencies]
 easy_reader = "0.5"
 glob = "0.3"
```

Comment:
I would also consider to move the version to the workspace even for those dependencies used only once.

Reply:
fixed